WcaOnRails/db/migrate/20220529123854_email_default_change.rb

```diff
@@ -0,0 +1,7 @@
+class EmailDefaultChange < ActiveRecord::Migration[6.1]
```

Comment:
Also, see other migrations for a `frozen_string_literal` comment that has to be at the top of the file, otherwise RuboCop starts crying like a baby...